Week1/homework/js-exercises/5-roundNumber.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const z = 7.25;
+
+console.log(z);
+
+const a = 7 ;
+console.log(a);
+
+if (a < z) {
+    const b = z;
+    console.log('The value of the variable Z is the highest');
+    console.log(b);
+}else{
+    const b = a;
+    console.log('The value of the variable A is the highest');
+    console.log(b);
+};
```

Comment:
Ah nice! You can also do this in an easier way with [Math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max)